### ISSUE TITLE
Insert self-linking QR Code

### ISSUE THREAD

**Issue by bryceschober** [guoguo12/url-shortener#3]
It sure would be helpful if there was a shortcut to create a goo.gl shortlink to the document itself. The user story would basically be:

As a google docs user, I would like every document I to print a qr code link to view the electronic version of itself.

**Comment by guoguo12** [comment]
Thanks for the suggestion! So basically, you'd like to be able to get QR codes in addition to regular shortlinks?

(This might actually be feasible, since you can apparently append `.qr` to any `goo.gl` shortlink to get a QR code.)

**Comment by bryceschober** [comment]
Yep, exactly. That way when we print shared documents and post them around the office, people could know exactly how to find them.
